Python/peephole.c

```diff
@@ -190,14 +141,23 @@ fold_tuple_on_constants(_Py_CODEUNIT *codestr, Py_ssize_t c_start,
     if (newconst == NULL) {
         return -1;
     }
-    for (i=0 ; i<n ; i++) {
-        constant = objs[i];
+
+    Py_ssize_t len_consts = PyList_GET_SIZE(consts);
```

Comment:
Why this have been moved? It is better to get a size just before calling `PyList_Append()` as in previous code.